docroot/themes/humsci/su_humsci_theme/js/hover_menu.js

```diff
@@ -29,25 +29,67 @@
         }
       }
 
+      // Check main menu items to ensure dropdown submenus don't get orphaned or cut off.
+      function menuEdgeCheck() {
+        var viewportWidth = document.body.clientWidth;
+        jQuery('ul.decanter-nav-primary > li').each(function() {
```

Comment:
should be able to use `$` instead of `jQuery` here.

Reply:
ok done